remote_attestation/rust/build.rs

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    oak_utils::compile_protos(&["../proto/remote_attestation.proto"], &["../proto"]);
```

Comment:
This adds derive `HandleVisit` annotation for protobuf messages in `remote_attestation.proto`. We don't need that. Could you replace this with `prost_build::compile_proto(...)` or `oak_utils::compile_protos_with_option` and passing a `ProtoOptions` with `derive_handle_visit` set to false, and try again without adding a dependency to `oak_io`?

Reply:
Thanks!
Done